ci_framework/roles/openshift_provisioner_node/tasks/config_dhcpd.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install DHCP server
+  become: true
+  ansible.builtin.package:
+    name: dhcp-server
+    state: present
+
+- name: Copy the dhcp conf
```

Comment:
I think that one plus the one bellow may be a "bootstrap" step - adding the `bootstrap` tag would help getting things ready beforehand.

Reply:
done.